superbench/common/utils/topo_aware.py

```diff
@@ -47,8 +76,16 @@ def gen_topo_aware_config(host_list, ibstat_file, ibnetdiscover_file, min_dist,
         topology distance (#hops).
     """
     config = []
-    if not ibstat_file or not ibnetdiscover_file:
-        logger.error('Either ibstat or ibnetdiscover not specified.')
+    if not Path.exists(ibstat_file):
```

Comment:
Should we also consider None as `ibstat_file`?

Reply:
Good catch, have revised